app/scripts/services/secrets.js

```diff
@@ -28,17 +28,29 @@ angular.module("openshiftConsole")
       return secretsByType;
     };
 
+    var setDataParams = function(decodedSecretData, serverName, data) {
+      var usernamePassword = [];
+      if (data.auth) {
+        usernamePassword = window.atob(data.auth).split(":");
```

Comment:
We should catch exceptions to avoid a runtime error if it's not properly encoded.

https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/atob